crates/rust-project-goals-cli/src/rfc.rs

```diff
@@ -419,6 +442,12 @@ impl Display for GithubAction<'_> {
             GithubAction::CreateIssue { issue } => {
                 write!(f, "create issue \"{}\"", issue.title)
             }
+            GithubAction::ChangeMilestone { number, milestone } => {
+                write!(f, "change issue #{} milestone \"{}\"", number, milestone)
+            }
+            GithubAction::Comment { number, body: _ } => {
+                write!(f, "comment on issue #{}", number)
```

Comment:
```suggestion
                write!(f, "post comment on issue #{}", number)
```

I guess it could be good to clarify what kind of comment the tool would be posting, but currently there can only be comments about the new period, if I understand correctly. We could even say this here instead of a generic "post comment" action description, something like "post comment on a continuing goal's tracking issue about the new period" or something.